tools/cldr-apps/src/main/webapp/js/new/cldrGear.js

```diff
@@ -179,99 +179,100 @@ const cldrGear = (function () {
   let alreadySet = false;
 
   function set(gearMenuItems) {
-    if (!alreadySet) {
-      alreadySet = true;
```

Comment:
The only change in this file is to return early if alreadySet, and reduce the indentation for the rest of the function

Reply:
good, turned off whitespace for my review